src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/Http/FrameResponseStream.cs

```diff
@@ -31,6 +34,7 @@ public override long Position
             set => throw new NotSupportedException();
         }
 
+        // REVIEW: Should Flush() throw?
```

Comment:
It means people can't use `StreamWriter` over the body even if they wrote *everything* async.